woodwork/tests/accessor/test_statistics.py

```diff
@@ -108,10 +112,12 @@ def test_accessor_bin_numeric_cols_into_categories():
     assert data["ints1"].equals(pd.Series([0, 1, 3, 1], dtype="int8"))
     assert data["ints2"].equals(pd.Series([0, 1, 0, 1], dtype="int8"))
     assert data["ints3"].equals(pd.Series([0, 1, 3, 1], dtype="int8"))
+    assert data["ints4"].equals(pd.Series([0, 1, 1, 0], dtype="int8"))
```

Comment:
these are bools right but we are casting to int type? This will work for True / False representations and 0 / 1 representations?

Reply:
Yep, it handles both bool and int representations!